main.go

```diff
@@ -161,33 +167,26 @@ func SendWarningNotification(ctx context.Context, j *Job) error {
 		j.ResultFolder,
 	)
 
-	return sendNotif(ctx, j, j.Status, subject, msg)
+	return sendNotif(ctx, j, j.Status, subject, msg, "analysis_status_change")
 }
 
 func SendPeriodicNotification(ctx context.Context, j *Job) error {
-	starttime, err := time.ParseInLocation(TimestampFromDBFormat, j.StartDate, time.Local)
+	durString, starttime, err := getJobDuration(j)
 	if err != nil {
-		return errors.Wrapf(err, "failed to parse start date %s", j.StartDate)
+		return err
 	}
 
-	// just print H(HH):MM format
-	dur := time.Since(starttime).Round(time.Minute)
-	h := dur / time.Hour
-	dur -= h * time.Hour
-	m := dur / time.Minute
-	durString := fmt.Sprintf("%d:%02d", h, m)
-
 	subject := fmt.Sprintf(PeriodicSubjectFormat, j.Name, starttime, durString)
 
 	msg := fmt.Sprintf(
 		PeriodicMessageFormat,
 		j.Name,
 		j.ID,
 		starttime,
-		dur,
+		durString,
```

Comment:
This is a small secondary change -- the notification text in the DE was showing "0s" due to the way we were modifying the `dur` object. It should be showing the same thing as the subject line in the DE/as the email subject, so now it is.